format/index.md

```diff
@@ -589,9 +589,9 @@ first or last character:
 The following characters **MUST NOT** be used in member names:
 
 - U+002B PLUS SIGN, "+" _(used for ordering)_
-- U+002C COMMA, "," _(used separator for multiple relationship paths)_
-- U+002E PERIOD, "." _(used as relationship path separators)_
-- U+005B LEFT SQUARE BRACKET, "[" _(use in sparse fieldsets)_
+- U+002C COMMA, "," _(used as a separator between relationship paths)_
```

Comment:
To me, I think that's still ambiguous about separating *within* versus separating *between*. If you do "used to separate relationship paths" for comma, I guess you should do "used to separate relationship names within relationship paths" for period.